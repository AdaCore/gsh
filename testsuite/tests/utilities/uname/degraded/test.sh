echo "unknown option -q:"
uname -q

echo ""
echo "unexpected argument:"
uname with_an_arg

echo ""
echo "unexpected operand introduce by '--':"
uname --
