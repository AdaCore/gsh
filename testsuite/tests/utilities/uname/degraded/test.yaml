title: degraded Test for uname builtin
