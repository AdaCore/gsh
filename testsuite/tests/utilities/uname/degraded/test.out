unknown option -q:
uname: unknown option

unexpected argument:
uname: extra operand

unexpected operand introduce by '--':
uname: unexpected operand
